### PR TITLE
changed tokenizer interface

### PR DESCRIPTION
**Pull request by JensWehner**

### CODE REVIEW

**Comment by JensWehner** [comment]
@votca-bot format

**Comment by JensWehner** [comment]
@votca-bot changelog: adapted tokenizer api

File: src/libcsg/modules/io/lammpsdumpreader.cc [hunk 1]
```diff
@@ -161,7 +158,7 @@ void LAMMPSDumpReader::ReadAtoms(Topology &top, string itemline) {
 
   {
     tools::Tokenizer tok(itemline.substr(12), " ");
-    tok.ToVector(fields);
+    fields = tok.ToVector();
```
Comment (felipeZ) [inline src/libcsg/modules/io/lammpsdumpreader.cc:161]: should this `field` assignment be written in a single line together with the `fields` declaration?

**Review by felipeZ — APPROVED** [review]

File: src/libcsg/modules/io/lammpsdumpreader.cc (hunk 1 above)
Comment (JensWehner) [reply src/libcsg/modules/io/lammpsdumpreader.cc:161]: yeah fields is declared earlier. So I only wanted to change the tokenizer api and not everything else. I agree in principle, but I did not want to do a larger scale refactoring.

**Pull request merged**